nixos/modules/services/networking/gnunet.nix

```
@@ -155,7 +155,7 @@ in
       description = "GNUnet";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      restartTriggers = [ configFile ];
+      restartTriggers = [ "/etc/gnunet.conf" ];
```

Using `/etc/gnunet.conf` would never trigger restarts because it is a constant string and hence does not change this unit file.

```suggestion
      restartTriggers = [ config.environment.etc."gnunet.conf".source ];
```